src/components/DpIcon/util/iconVariables.js

```diff
@@ -165,6 +166,7 @@ const baseIcons = {
   square: {
     phosphorName: 'square',
     path: 'M208,32H48A16,16,0,0,0,32,48V208a16,16,0,0,0,16,16H208a16,16,0,0,0,16-16V48A16,16,0,0,0,208,32Zm0,176H48V48H208V208Z'
+
```

Comment:
Nitpick: unneeded linebreak.

Reply:
Removed in df50704a2ed7af1be087465640bb82627b312c9a